<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="modifier">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>641</width>
      <height>301</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>ajouter</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>20</y>
        <width>411</width>
        <height>221</height>
       </rect>
      </property>
      <property name="title">
       <string>ajouter commande</string>
      </property>
      <widget class="QComboBox" name="comboBox">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>20</y>
         <width>221</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>5</string>
        </property>
       </item>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>190</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>ajouter </string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>26</x>
         <y>20</y>
         <width>81</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>liste produit</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_prenom">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>80</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>80</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>prenom</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>170</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>quantite</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_quantite">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>170</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_nom">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>50</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>50</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>nom</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>110</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>adresse</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>140</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>mail</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_adresse">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>110</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_mail">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>140</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_9">
     <attribute name="title">
      <string>afficher</string>
     </attribute>
     <widget class="QTableView" name="table_commande">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>611</width>
        <height>261</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>modifier</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>20</y>
        <width>451</width>
        <height>241</height>
       </rect>
      </property>
      <property name="title">
       <string>modifier commande</string>
      </property>
      <widget class="QComboBox" name="comboBox_2">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>20</y>
         <width>221</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>5</string>
        </property>
       </item>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>190</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>modifier</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>26</x>
         <y>20</y>
         <width>81</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>liste produit</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>80</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>80</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>prenom</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>170</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>quantite</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_55">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>170</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_1">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>50</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_17">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>50</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>nom</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_18">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>110</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>adresse</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_19">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>140</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>mail</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_3">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>110</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_4">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>140</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_20">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>200</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>id</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_id4">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>200</y>
         <width>111</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_5">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>50</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_21">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>50</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>etat</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>supp</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>10</y>
        <width>341</width>
        <height>231</height>
       </rect>
      </property>
      <property name="title">
       <string>supprimer</string>
      </property>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>id_commande</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_3">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>120</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>supp</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_id3">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>40</y>
         <width>111</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>recherche</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>0</y>
        <width>341</width>
        <height>231</height>
       </rect>
      </property>
      <property name="title">
       <string>recherche</string>
      </property>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>cin</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="recherche">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>40</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_4">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>40</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>rechercher</string>
       </property>
      </widget>
      <widget class="QTableView" name="tableView_recherche">
       <property name="geometry">
        <rect>
         <x>5</x>
         <y>70</y>
         <width>331</width>
         <height>121</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>tri</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>571</width>
        <height>251</height>
       </rect>
      </property>
      <property name="title">
       <string>tri</string>
      </property>
      <widget class="QPushButton" name="tri">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>10</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>trie dec</string>
       </property>
      </widget>
      <widget class="QTableView" name="table_view_tri">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>40</y>
         <width>271</width>
         <height>181</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="tri_2">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>10</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>trier crois</string>
       </property>
      </widget>
      <widget class="QTableView" name="table_view_tri_2">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>40</y>
         <width>271</width>
         <height>181</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>stats</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>20</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>afficher stat</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_7">
     <attribute name="title">
      <string>facture</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>10</y>
        <width>451</width>
        <height>231</height>
       </rect>
      </property>
      <property name="title">
       <string>facture</string>
      </property>
      <widget class="QPushButton" name="pushButton_7">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>70</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>facture</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>70</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>id</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_id">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>70</y>
         <width>111</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_8">
     <attribute name="title">
      <string>annuler</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>70</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>ID_COMMANDE</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>70</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>annuler</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_id2">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>70</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
